VMs/metrics/2_build_baseimage.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# Copyright (c) 2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+source global_env.sh
+
+die() {
```

Comment:
These functions are defined in all the scripts, so how about putting them into a `lib.sh` and `source`-ing?

Reply:
yep, did occur to me - will do